Documentation/Troubleshooting/ceph-common-issues.md

```diff
@@ -507,6 +507,65 @@ You don't need to restart the pod, the effect will be immediate.
 
 To disable the logging on file, simply set `log_to_file` to `false`.
 
+## Collect coredump and perf data of a ceph process at runtime
```

Comment:
Does this section only discuss the perf collection? The core dump was documented in #11163, right? But if we want to discuss core dump collection, seems like a separate section would be better.
```suggestion
## Collect perf data of a ceph process at runtime
```

Reply:
here actually we are collecting both coredump and perf output of a live process together for troubleshooting. Does it make sense to club them together ?

Reply:
I see the `gdbpmp.data` is collected, so it seems focused just around the profiling. While the perf data does require the core dump, I'm thinking the header can just focus on the perf profiling as the reason for doing all these steps. If they only want the core dump, they don't have to do any of these steps, right?